src/main/resources/db/migration/V8.0__DDL_modify_contract_table.sql

```diff
@@ -0,0 +1,31 @@
+ALTER TABLE contract ADD COLUMN active BOOLEAN DEFAULT FALSE;
+
+CREATE OR REPLACE FUNCTION on_modify_update_modified_at()
```

Comment:
I'm not fun of set business login as a trigger in the db. But what do you think? @eazaretzky @FacuOrellana @maxituc21 @marcosihe